demo/main.go

```diff
@@ -3,6 +3,8 @@
 
 package main
 
+import _ "github.com/open-telemetry/opentelemetry-go-compile-instrumentation/instrumentation/helloworld"
```

Comment:
I would add comment for now
```suggestion
// This dependency is added manually for now, as refreshDeps is not yet implemented. TODO: remove it once implemented.
import _ "github.com/open-telemetry/opentelemetry-go-compile-instrumentation/instrumentation/helloworld"
```